components/backup/base/external-secret.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: CHANGE_ME
```

Comment:
Since we create one secret per cluster, it think it would be better to have the same name for the secret in all cluster, this will be simpler to configure adp operator.

Reply:
ok, the name now the same for all clusters but let's find a better name :) . What is the secret, bucket key? give it a name that describe what is this secret

Reply:
You change the secret name that will be created, let's change the external secret name as well, we usually use the same name so replace `CHANGE_ME` by `backup-s3-credentials`